blocks/components/editable/format-toolbar.js

```diff
@@ -101,29 +103,31 @@ class FormatToolbar extends wp.element.Component {
 	}
 
 	render() {
-		const { formats, focusPosition } = this.props;
+		const { formats, focusPosition, enabledControls = DEFAULT_CONTROLS } = this.props;
 		const linkStyle = focusPosition
 			? { position: 'absolute', ...focusPosition }
 			: null;
 
+		const toolbarControls = FORMATTING_CONTROLS
+			.filter( control => enabledControls.indexOf( control.format ) !== -1 )
+			.map( ( control ) => ( {
+				...control,
+				onClick: this.toggleFormat( control.format ),
+				isActive: !! formats[ control.format ]
+			} ) );
+
+		if ( enabledControls.indexOf( 'link' ) !== -1 ) {
```

Comment:
Because it has a different behaviour (different `onClick` and  `isActive` props)